plugin-server/src/main/graphile-worker/graphile-worker.ts

```diff
@@ -80,10 +81,7 @@ export class GraphileWorker {
             enqueueFn = () =>
                 runRetriableFunction({
                     hub: this.hub,
-                    metricName: 'job_queues_enqueue',
-                    metricTags: {
-                        jobName,
-                    },
```

Comment:
This might look suspicious, but `runRetriableFunction` is _only_ used here, and carrying this `metricsTags` baggage into a Prometheus (label based, sensitive to high cardinality) world isn't worth it.